tfhe/src/lib.rs

```diff
@@ -69,3 +69,7 @@ pub mod safe_deserialization;
 pub mod conformance;
 
 pub mod named;
+
+#[cfg(feature = "forward_compatibility")]
+/// cbindgen:ignore
+pub mod forward_compatibility;
```

Comment:
Could be part of a separate commit

Reply:
not really wanting to hunt down to make commits that are self consistent